### ISSUE TITLE
npm install -g fails with `--ignore-scripts`

### ISSUE THREAD

**Issue by rtfeldman** [avh4/elm-format#378]
```
npm install -g elm-format@exp --ignore-scripts
```

```
elm-format --version
events.js:163
      throw er; // Unhandled 'error' event
      ^

Error: spawn /usr/local/lib/node_modules/elm-format/unpacked_bin/elm-format ENOENT
    at exports._errnoException (util.js:1050:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:367:16)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
    at Module.runMain (module.js:607:11)
    at run (bootstrap_node.js:427:7)
    at startup (bootstrap_node.js:151:9)
    at bootstrap_node.js:542:3
```

Potentially relevant info:

```
node --version
v7.10.0
```

```
npm --version
4.2.0
```

![screen shot 2017-06-12 at 10 13 27 am](https://user-images.githubusercontent.com/1094080/27046085-02ba6e30-4f58-11e7-9e72-6d86bf7f2e6b.png)

**Comment by avh4** [comment]
This needs to be fixed in https://github.com/avh4/binwrap/issues/7

**Comment by avh4** [comment]
This is fixed in the 0.8.0-rc3 release  `npm install -g elm-format@rc --ignore-scripts`
